.github/llms.md

```diff
@@ -0,0 +1,38 @@
+# Extension Pack for Java
```

Comment:
this is the extension pack description, is this expected for spring-boot-dashboard extension?